Demos/Makefile

```diff
@@ -6,7 +6,15 @@ test:	all
 	python run_numeric_demo.py
 	python run_spam.py
 	cd callback; $(MAKE) test
-
+	cd embed; $(MAKE) test
+	#
+	# longer running
+	#python integrate_timing.py
+	# also  run
+	# python setup.py build_ext --inplace in
+	#  benchmarks to build benchmarks
+	# python setup.py build_ext --inplace in libraries then
```

Comment:
No, I meant "copy" as in "users would want to copy&paste this", which is easier if you can copy a whole line rather than having to know where to stop. Specifically, executing `python setup.py build_ext --inplace in` is not going to work.